scripts/mkimage

```diff
@@ -263,6 +263,14 @@ EOF
     mcopy $RELEASE_DIR/target/KERNEL.md5 "::/$KERNEL_NAME.md5"
     mcopy $RELEASE_DIR/target/SYSTEM.md5 ::/SYSTEM.md5
 
+    # compile and copy amlogic autoscript files to part1
```

Comment:
You build u-boot:host only for S905: https://github.com/LibreELEC/LibreELEC.tv/pull/2315/files#diff-9c7eee3ed0931e5ee6bcba810283ece0R185
So I suggest to check for PROJECT == S905 here too.